e2e_tests/integration/data-export.spec.js

```diff
@@ -31,17 +31,68 @@ describe('Data export', () => {
     const password = Cypress.config('password')
     cy.connect('neo4j', password)
   })
-  it('shows export buttons on viz load', () => {
+  it('shows the correct export buttons', () => {
     cy.executeCommand(':clear')
     cy.executeCommand('CREATE (n:ExportTest) RETURN n')
 
     cy.get('[data-testid="frame"]', { timeout: 10000 }).should('have.length', 1)
-    cy.get('[data-testid="frame-export-dropdown"]')
-      .first()
-      .trigger('mouseover')
-    cy.get('[data-testid="frame-export-dropdown"]')
-      .first()
-      .should('contain', 'Export PNG')
+
+    // viz
```

Comment:
Since these blocks (viz, table, text, code) are testing exactly in the same way, you could [generate these tests dynamically](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Dynamically-Generate-Tests) making the file less verbose and making it obvious that they're doing the same thing.